Lib/test/test_import/__init__.py

```diff
@@ -2330,6 +2330,9 @@ def test_variants(self):
             self.assertIs(basic.look_up_self(), basic_lookedup)
             self.assertEqual(basic.initialized_count(), expected_init_count)
 
+            loaded.module._clear_module_state()
```

Comment:
Why not use [addCleanup](https://docs.python.org/3/library/unittest.html#unittest.TestCase.addCleanup)?